manifests/m/Microsoft/VisualStudio/2010/ToolsOfficeRuntime/60828.00/Microsoft.VisualStudio.2010.ToolsOfficeRuntime.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created with Komac v1.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2010.ToolsOfficeRuntime
+PackageVersion: '60828.00'
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://www.microsoft.com
+PackageName: Visual Studio 2010 Tools for Office Runtime

```

```suggestion
PackageName: Microsoft Visual Studio 2010 Tools for Office Runtime
```
This should match control panel as closely as possible